app/models/email.rb

```diff
@@ -5,6 +5,14 @@ def self.create_from_params!(params)
     build_from_params(params).tap(&:save!)
   end
 
+  def self.create_from_subscription_content!(params, subscription_content)
+    create_from_params!(
+      params.merge(
+        address: subscription_content.subscription.subscriber.address
```

Comment:
This couples things really deeply. If all we need is the address would it be better to just add that to the params and use `create_from_params`?

Reply:
Yep, agreed. That should be fixed now!